src/utilities/utilities.js

```diff
@@ -54,3 +54,60 @@ export const addAccessibility = (children, slidesToShow, currentSlide) => {
     });
   }
 };
+
+export const getPropsByTransitionMode = (props, keys) => {
+  const { slidesToShow, transitionMode } = props;
```

Comment:
you can destructure these props in the function declaration like so: ```export const getPropsByTransitionMode = ({slidesToShow, transitionMode}, keys) => {...```

Reply:
if we only need those properties, it's clearer for the reader to see which ones we need explicitly, rather than getting all of props

Reply:
Awesome. Will do.

Reply:
Moving the destructuring to the function declaration won't work for the `default` and `else` statement of the function. I tried a couple of different things but it always break the function.

Reply:
Ah I see - yes, we'll need the whole props object to access the key properties on it. so we won't be able to destructure the properties we need from props in the first line. good catch!